Jenkinsfile

```diff
@@ -1,136 +1,217 @@
-                                         description: 'All unit tests passed',
-                                         targetUrl: "${env.JOB_URL}/workflow-stage")
-                    }
-                } catch(Exception ex) {
-                    if (env.CHANGE_ID) {
-                        pullRequest.createStatus(status: 'failure',
-                                     context: 'jenkins/unittests',
-                                     description: 'unit tests failed',
-                                     targetUrl: "${env.JOB_URL}/workflow-stage")
-                    }
-                    currentBuild.result = 'UNSTABLE'
-                }
-            }
-        }
-    }
-    stage("lint test-cases") {
-        steps {
-            script {
-                try {
-                    sh ''' ./utils/lint_test_cases.sh '''
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 
-                    if (env.CHANGE_ID) {
-                        pullRequest.createStatus(status: 'success',
-                                         context: 'jenkins/lint_test_cases',
-                                         description: 'all test cases are checked',
-                                         targetUrl: "${env.JOB_URL}/workflow-stage")
-                    }
-                } catch(Exception ex) {
-                    if (env.CHANGE_ID) {
-                        pullRequest.createStatus(status: 'failure',
-                                         context: 'jenkins/lint_test_cases',
-                                         description: 'some test cases failed to check',
-                                         targetUrl: "${env.JOB_URL}/workflow-stage")
-                    }
-                    currentBuild.result = 'UNSTABLE'
-                }
-            }
-        }
-    }
-    stage("provision test") {
-        when { changeRequest() }
-        steps {
-            node('aws-sct-builders-eu-west-1') {
-                script {
-                    def labels = []
-                    pullRequest.labels.each { labels.add("${it}") }
-                    def should_test = labels.find { it == "test-provision" }
+def target_backends = ['aws', 'gce']
+
+def runSctProvisionTest(String backend){
+	if (currentBuild.result != null) {
+		return
+	}
+	echo "Going to Provision test on ${backend} for Pull Request ID: ${env.CHANGE_ID}"
+	checkout scm
+	def distro_cmd = ""
+	if (backend == 'gce') {
+		distro_cmd = "export SCT_SCYLLA_LINUX_DISTRO='centos'"
+	}
+	sh """
+	#!/bin/bash
+	set -xe
+	env
+	export SCT_CLUSTER_BACKEND="${backend}"
+	export SCT_COLLECT_LOGS=false
+	export SCT_POST_BEHAVIOR_DB_NODES="destroy"
+	export SCT_POST_BEHAVIOR_LOADER_NODES="destroy"
+	export SCT_POST_BEHAVIOR_MONITOR_NODES="destroy"
+	export SCT_INSTANCE_PROVISION="on_demand"
+    export SCT_CONFIG_FILES="test-cases/PR-provision-test.yaml1"
```

Comment:
typo in the filename: `test-cases/PR-provision-test.yaml1` -> `test-cases/PR-provision-test.yaml`

Reply:
Fixed